docs/reference/troubleshooting.md

```diff
@@ -0,0 +1,691 @@
+
+        cross-seed:
+            network_mode: "service:vpn"
+    ```
+
+### Config-Related Issues
+
+#### Windows Path Issues
+
+**Common causes:**
+
+- Single backslashes causing escape character problems
+- Path format inconsistencies
+
+**Solutions:** Use double backslashes in all Windows paths:
+
+```js
+// ✗ INCORRECT
+linkDirs: ["C:\Users\Name\Downloads\links"],
+
+// ✓ CORRECT
+linkDirs: ["C:\\Users\\Name\\Downloads\\links"],
+```
+
+#### Unraid Path Issues
+
+**Common causes:**
+
+- Using cache paths instead of user share paths
+- Permission problems with user/group IDs
+
+**Solutions:**
+
+1. Use user share paths in your config:
+    ```js
+    linkDirs: ["/mnt/user/data/torrents/links"],
+    ```
+2. Set correct permissions in the container:
+    ```yaml
+    environment:
+        - PUID=99 # Your Unraid user ID
+        - PGID=100 # Your Unraid group ID
+    ```
+
+## Log Interpretation Guide
+
+### Common Log Messages
+
+#### "[prefilter] skipping searchee: ..."
+
+This message explains why a torrent was excluded from searching.
+
+- **excluded by size**: The torrent falls below minSize or above maxSize
+- **excluded by content type**: Non-video content with includeNonVideos: false
+- **excluded by age**: Matches excludeOlder setting
+- **excluded by recent search**: Matches excludeRecentSearch setting
+- **excluded by blocklist**: Matches an entry in your blocklist
+
+#### "[match] rejected: ..."
+
+This explains why a potential match was rejected.
+
+- **it has a different file tree**: File structure doesn't match
+- **it's too different in size**: Size difference exceeds fuzzySizeThreshold
+- **it has the same infohash**: Already have this exact torrent
+- **error accessing data**: Problems accessing the files
+
+#### "[inject] failed: ..."
+
+These messages indicate injection problems.
+
+- **client error**: The torrent client returned an error
+- **cannot find linkDir on same filesystem**: Hardlinking failed
+- **torrent already exists in client**: Already added
+
+## Troubleshooting Flowcharts
+
+### Startup Issues Flowchart
+
+```mermaid
+flowchart TD
+    Start([Start]) --> A{cross-seed starts without errors?}
+    A -->|Yes| Done([Done])
+
+    A -->|No| B{What error?}
+
+    B -->|SyntaxError in config.js| C[Check for: missing commas, quotes, brackets]
+    C --> Fix1[Fix syntax errors in config.js]
+    Fix1 --> Start
+
+    B -->|RangeError: WebAssembly.instantiate()| D[Set NODE_OPTIONS=--disable-wasm-trap-handler]
+    D --> Start
+
+    B -->|val is not a non-empty string| E[Delete config.db]
+    E --> Start
+
+    B -->|EADDRINUSE| F[Change port or kill existing process]
+    F --> Start
+
+    classDef process fill:#f0f0f0,stroke:#333,stroke-width:1px
+    classDef decision fill:#e1f5fe,stroke:#0277bd,stroke-width:1px
+    classDef endpoint fill:#e8f5e9,stroke:#2e7d32,stroke-width:1px
+
+    class Start,Done endpoint
+    class A,B decision
+    class C,D,E,F,Fix1 process
+```
+
+### Client Connection Flowchart
```

Comment:
these flowcharts seem like they're just repeating what was said earlier. we should pick one or the other, probably the text, unless the flowchart is really valuable.